### PR TITLE
Specify path to python3.5 for perf jobs

### PR DESCRIPTION
**Pull request by adiaaida**

### CODE REVIEW

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

**Comment by adiaaida** [comment]
@DrewScoggins PTAL. This fixes the issues we are seeing with https://github.com/dotnet/core-eng/issues/4400

File: perf.groovy [hunk 1]
```diff
@@ -36,11 +36,13 @@ def static getOSGroup(def os) {
                         def architecture = arch
                         def jobName = isSmoketest ? "perf_perflab_${os}_${arch}_${opt_level}_${jit}_smoketest" : "perf_perflab_${os}_${arch}_${opt_level}_${jit}"
                         def testEnv = ""
+                        def python = "C:\\Python35\\python.exe"
 
                         def newJob = job(Utilities.getFullJobName(project, jobName, isPR)) {
                             // Set the label.
                             if (isSmoketest) {
                                 label('Windows.Amd64.ClientRS4.DevEx.15.8.Perf')
+                                python = "C:\\python3.7.0\\python.exe"
```
Comment (maririos) [inline perf.groovy]: Out of curiosity, us there a reason why for the smoke tests we use another version?

Comment (adiaaida) [reply perf.groovy]: When I created the helix queue, that was the latest version (and easiest installer to get). When we reinstall the perfsnakes, we'll likely want to upgrade their python install as well.

**Comment by MattGal** [comment]
I have a simpler fix I'm going to discuss w/ Mariana.

**Comment by adiaaida** [comment]
@MattGal Thanks! I've made the change and if it works will close this PR.

**Comment by adiaaida** [comment]
@dotnet-bot test this please

**Comment by adiaaida** [comment]
I've reopened this issue because we continue to see the python 103 error, despite adding the py.ini file to the machines. It seems to be a sporadic issue that pops up on random machines at random times, with the only fix being to log on to the machines, which is not a good use of our time. So this change is now needed.

**Comment by DrewScoggins** [comment]
LGTM

**Comment by adiaaida** [comment]
@RussKeldorph I'm I free to ignore the dotnet-coreclr job for now?

**Comment by RussKeldorph** [comment]
@adiaaida Yes, we appear to have infra issues with that.

@morganbr @sbomer @hoyosjs

**Comment by sbomer** [comment]
My apologies - I just disabled it for PRs.

**Pull request merged**